include/seqan/blast/blast_statistics.h

```diff
@@ -555,6 +573,35 @@ _selectSet(BlastScoringScheme<TScore> & scheme)
     return false;
 }
 
+template <typename TValue, typename TSpec>
+inline bool
+_selectSet(BlastScoringScheme<Score<TValue, ScoreMatrix<AminoAcid, TSpec>>> & scheme)
+{
+    using TScore = Score<TValue, ScoreMatrix<AminoAcid, TSpec>>;
+    using TKAValues = KarlinAltschulValues<TScore>;
+    return _selectSet<TScore, TKAValues>(scheme);
+}
+
+template <typename TValue>
+inline bool
+_selectSet(BlastScoringScheme<Score<TValue, ScoreMatrix<AminoAcid, ScoreSpecSelectable>>> & scheme)
```

Comment:
Ok, so if in the complete file everything is in this style, than leave it as is.
I also agree, that if we start a new module, within that module the scheme can be modernised, but when adding something to existing modules, those should stay consistent.